src/System.Private.CoreLib/src/System/Threading/LazyInitializer.cs

```diff
@@ -86,17 +78,12 @@ public static class LazyInitializer
         /// if an object was not used and to then dispose of the object appropriately.
         /// </para>
         /// </remarks>
-        public static T EnsureInitialized<T>(ref T target, Func<T> valueFactory) where T : class
-        {
-            // Fast path.
+        public static T EnsureInitialized<T>(ref T target, Func<T> valueFactory) where T : class =>
             // According to the abstract ECMA CLI memory model, this read should be volatile, due to possible reorderings with
             // subsequent reads from fields of "target."  However, as of today (7/12/12), Volatile.Read<T> is quite slow, and in practice
             // this read does *not* need to be volatile, because our memory model preserves ordering of data-dependent reads.
-            if (target != null)
-                return target;
-
-            return EnsureInitializedCore<T>(ref target, valueFactory);
-        }
+            // Another occurrence at line 183.
```

Comment:
Thanks @jkotas, not being very sure, I thought I would check as there was mix of styles. I have made changes in both the places.